.gitignore

```diff
@@ -35,7 +35,6 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
-package-lock.json
```

Comment:
recommended practice is to commit package-lock.json

Reply:
For software yes, but for a specification? There I'd assume you'd want the latest version?